### PR TITLE
implement #77 ifZero and add isOdd

### PR DESCRIPTION
**Pull request by moogacs**
### Details
**Link to the issue:** *link to the issue*
https://github.com/rgehan/hacktoberfest-2k18-katas/issues/77
**Name of the function you implemented**: *name of the function*
ifzero
**Name of the function you added test cases for**: *name of the new function*
none
### Checklist
- [ x ] I have asked for this issue, and was assigned it
- [ x ] I have properly filled the "Details" section above  :arrow_up:
- [ x ] I didn't modify any test case  :red_circle:
- [ x ] I have implemented the function and all its tests pass  :white_check_mark:
- [x] I have added a new test case for someone to implement  :new:
- [ ] I have created an issue for my new test case  :speak_no_evil:
- [ x ] I've :star: the repository (or not, all up to you)
- [ x ] And more importantly, I've had fun! :beer:

### CODE REVIEW

**Comment by rgehan** [comment]
You're not supposed to implement yourself the kata you added, remove the implementation of `isOdd`

File: yarn.lock [hunk 1]
```diff
@@ -1,3622 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (TimothePearce) [inline yarn.lock]: Why deleting all the content ?

Comment (moogacs) [reply yarn.lock]: I didn't get it so you just need me to keep isOdd.test and the implementation
 for it will let it to someone else

Comment (rgehan) [reply yarn.lock]: Keep `ifZero.js`, and also add:
- `isOdd.test.js` with actual tests
- `isOdd.js` which should be EMPTY

And please leave `yarn.lock` alone

Comment (moogacs) [reply yarn.lock]: it has to be done now or shall I rebase and do it again ? @rgehan

**Comment by rgehan** [comment]
Where is your implementation now?

**Comment by rgehan** [comment]
Thank you. Could you create an issue for you new `isOdd` kata please?

**Pull request merged**